lib/wallet/seed.dart

```diff
@@ -5,7 +5,7 @@ import 'package:provider/provider.dart';
 import 'package:ten_ten_one/models/seed_backup_model.dart';
 import 'package:go_router/go_router.dart';
 
-import 'ffi.io.dart' if (dart.library.html) 'ffi.web.dart';
+import '../ffi.io.dart' if (dart.library.html) 'ffi.web.dart';
```

Comment:
🔧 I guess these kind of changes are the reasons for the dart style guideline not to use relative imports. 🙂

Change to `import 'package:ten_ten_one/ffi.io.dart' if (dart.libray.html) 'ffi.web.dart';`